.github/actions/vtk-install-dep/action.yml

```diff
@@ -131,7 +131,6 @@ runs:
         -DVTK_MODULE_ENABLE_VTK_RenderingAnnotation=YES
         -DVTK_MODULE_ENABLE_VTK_RenderingCore=YES
         -DVTK_MODULE_ENABLE_VTK_RenderingExternal=YES
-        -DVTK_MODULE_ENABLE_VTK_RenderingLabel=YES
```

Comment:
?

Reply:
Updating the Docker image was causing an issue with `octree.txx` in VTK used by `RenderingLabel`. Since we are not using it I decided to just skip building this module. It's fixed by https://gitlab.kitware.com/vtk/vtk/-/merge_requests/11178 but I do not see the point of bringing back an unused module.  
By the way, the CI is failing because of one other issue fixed by the same PR so I need to wait for the merge and bump the VTK version again.

Reply:
I wonder why we added it initially though. Well if CI is happy, its fine.